lib/services/project-data-service.ts

```diff
@@ -125,7 +125,7 @@ export class ProjectDataService implements IProjectDataService {
 
 	private async getIOSAssetSubGroup(dirPath: string): Promise<IAssetSubGroup> {
 		const pathToContentJson = path.join(dirPath, AssetConstants.iOSResourcesFileName);
-		const content = <IAssetSubGroup>this.$fs.readJson(pathToContentJson);
+		const content = this.$fs.exists(pathToContentJson) && <IAssetSubGroup>this.$fs.readJson(pathToContentJson) || { images: [] };
```

Comment:
What will happen in case when `pathToContentJson` exists but the file is not a valid json? I'm expecting to throw an error in this case.

Reply:
Yeah, I prefer to throw the error in this case as it is not expected to have a non-json file here.